modules/items/item-ffg.js

```diff
@@ -128,7 +128,7 @@ export class ItemFFG extends ItemBaseFFG {
           if (this.actor.type !== "vehicle" && this.actor.data.type !== "vehicle") {
             if (ModifierHelpers.applyBrawnToDamage(data)) {
               const olddamage = data.damage.value;
-              data.damage.value = parseInt(actorData.data.characteristics.Brawn.value, 10) + damageAdd;
+              data.damage.value = parseInt(actorData.data.characteristics[data.characteristic.value].value, 10) + damageAdd;
```

Comment:
Do we run the risk of nulls causing math errors if `data.characteristic.value` is null, undefined or a blank string?

Reply:
It has to pass through this check to be true. I think I came across that issue so I added the undefined check. Having a misspelled characteristic could possibly be an issue though but would be unsure how they would change it as it is selected from their Characteristic dropdown.
`
if(data.characteristic?.value !== "" && data.characteristic?.value !== undefined)
`

Reply:
Ah yep that's fine, didn't spot that it was inside the applyBrawnToDamage check. 👍